packages/core/src/actions.ts

```diff
@@ -69,6 +69,26 @@ export function resolveActionObject(
         return [state, a];
       }
     );
+  } else if (isArray(dereferencedAction)) {
+    const params = actionObject.params ?? {};
+    return createDynamicAction(
+      { type: ActionTypes.ActionGroup, params },
+      (_event, { state }) => {
+        const action: BaseActionObject = {
+          type: actionObject.type,
+          params: {
+            actions: toActionObjects(dereferencedAction).map((object) => ({
+              ...object,
+              params: {
+                ...object.params,
+                ...params
+              }
+            }))
+          }
```

Comment:
Since each action needs to receive `params`, I just resolved the action objects for each here and then mapped them to include the params.

Is that fine?

Reply:
Seems fine to me 👍